README.md

```diff
@@ -170,6 +170,14 @@ run tests-trigger with `--allow`:
 Of course, you should make sure that the pull request is proper and
 doesn't execute evil code during tests.
 
+### tests-trigger with a different origin
+
+If you need to specify --repo in tests-trigger as your remote is different from
+cockpit-project/cockpit you can set a git configuration option from which
+tests-trigger reads the repo. This has to be set per cockpit project.
+
+    $ git config cockpit.bots.github-repo cockpit-project/cockpit
```

Comment:
I find it really evil to have these `$` everywhere, as they break cut & paste. See https://github.com/cockpit-project/cockpit/commit/fe906e6bc92468ebf130d348d7fd1a7edc008c94 . While you are at it, would you mind dropping them everywhere in that file?